README.md

```diff
@@ -8,10 +8,11 @@ Install the requirements under `tf/requirements.txt`. And call `./init.sh` to co
 
 ## Data preparation
 
-In order to start a training session you first need to download trainingdata from http://lczero.org/training_data. This data is packed in tar.gz balls each containing 10'000 games or chunks as we call them. Preparing data requires the following steps:
+In order to start a training session you first need to download trainingdata from https://storage.lczero.org/files/training_data/. This data is packed in tar.gz balls each containing 10'000 games or chunks as we call them. Preparing data requires the following steps:
```

Comment:
This part is out of date as well.  We have tars of compressed training files that are one hours worth of training data each.
Since they are already compressed inside the tar the instructions below are also wrong, there is no need to run parallel gzip.

Reply:
Thanks, has just updated the instruction.